doc/modules/mixture.rst

```diff
@@ -195,6 +168,44 @@ to some mixture components getting almost all the points while most
 mixture components will be centered on just a few of the remaining
 points.
 
+.. figure:: ../auto_examples/mixture/images/plot_bayesian_gaussian_mixture_001.png
```

Comment:
This image URL is broken since the introduction of the sphinx gallery module in master: it should be:

 ../auto_examples/mixture/images/sphx_glr_plot_bayesian_gaussian_mixture_001.png